portal/server/helpers/date.ts

```diff
@@ -5,6 +5,5 @@
  * e.g. A Unix timestamp of 1702900314 is equivalent to an ISO 8601 date time
  * stamp of '2023-12-18T11:51:54Z'
  * 
- * @returns {number}
```

Comment:
Should this not stay?

Reply:
The return type is already captured by the TS. Adding this will add a second source of truth.
I could add a description of what is being returned but I felt the functions name was sufficient. Happy to be overridden here